plugin/db/util/mask_sensitive_data.go

```diff
@@ -185,22 +186,37 @@ func extractFieldName(in *tidbast.SelectField) string {
 	return ""
 }
 
+func (extractor *sensitiveFieldExtractor) checkFieldSensitive(databaseName string, tableName string, fieldName string) bool {
+	for i := len(extractor.outerSchemaInfo) - 1; i >= 0; i-- {
```

Comment:
Add a comment to explain why we loop the slice in reversed order?

Reply:
Added.